MirServer/ubuntu-frame
# Add a --wallpaper-enabled argument to optionally disable wallpaper draws

## To test
See the wallpaper:
```
./frame
./frame --wallpaper-enabled=true 
```

Don't see the wallpaper:
```
./frame --wallpaper-enabled=false
```

## What's new?
- Added a `wallpaper_enabled` flag that is set to true by default
- Only doing things in `BackgroundClient::Self::draw_screen` if we have errors or the wallpaper is enabled
- If we have an error _and_ we don't want the wallpaper _but_ we don't have a diagnostic file, then we draw the wallpaper as we did before (this may be undesirable, but its a very rare case, and it maintains previous behavior)
